services/graphite/service.go

```diff
@@ -334,6 +335,13 @@ func (s *Service) handleLine(line string) {
 	// Parse it.
 	point, err := s.parser.Parse(line)
 	if err != nil {
+		switch err := err.(type) {
+		case *ErrUnsupportedValue:
+			// Graphite ignores NaN values with no error.
+			if math.IsNaN(err.Value) {
```

Comment:
Let's count these, so we have visibility into how many points InfluxDB is dropping due to `IsNan` values. I suggest `statPointsNanFail`, with the text `pointsNanFail`.